### PR TITLE
Cisco SB/CBS environment sensors

### PR DESCRIPTION
**Pull request by Fehler12**
Added detection for environment sensors (FAN and PSU) for cisco small business devices

Pduring development I have encountered an error with the newer CBS models, which are also covered by this: the status for POE-usage and locked stats are only polled for the first ten interfaces: https://community.librenms.org/t/cbs250-snmp-does-not-fetch-all-values/19235

I'm pretty sure that this has nothing todo with this changes but I'm not 100% sure because I cannot find any reason.

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [ ] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by Fehler12** [comment]
PS: I did not add any test data because currently the generated test data produce false test entries because of the bug mentioned above.

**Comment by murrant** [comment]
There still needs to be test data.  Add it and if any error show up, we can help you resolve them.

**Comment by Fehler12** [comment]
I've added test data but with false results for the POE detection.

File: tests/data/ciscosb_cbs250-24p-4x.json [hunk 1]
```diff
@@ -0,0 +1 @@
+null
```
Comment (murrant) [inline tests/data/ciscosb_cbs250-24p-4x.json:1]: ???

**Pull request merged**